ytypes/leaf_test.go

```diff
@@ -1318,3 +1313,130 @@ func TestUnmarshalLeafRef(t *testing.T) {
 		}
 	}
 }
+
+func TestStripPrefix(t *testing.T) {
+	tests := []struct {
+		name     string
+		inName   string
+		wantName string
+		wantErr  string
+	}{{
+		name:     "valid with prefix",
+		inName:   "one:two",
+		wantName: "two",
+	}, {
+		name:     "valid without prefix",
+		inName:   "two",
+		wantName: "two",
+	}, {
+		name:    "invalid input",
+		inName:  "foo:bar:foo",
+		wantErr: "path element did not form a valid name (name, prefix:name): foo:bar:foo",
+	}}
```

Comment:
suggest adding empty string check.

Reply:
This isn't an error thus far - but check added.